samples/SqlServerCacheConcurencySample/project.json

```diff
@@ -11,6 +11,10 @@
     "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*"
   },
 
```

Comment:
Can you make it consistent with the others? (You removed an empty line from the other project.json below.)